k8s/hamm/hamm-cost-updater-deployment.yaml.ctmpl

```diff
@@ -80,6 +94,20 @@ spec:
           volumeMounts:
             - mountPath: /init_data
               name: appdir
+        # https://cloud.google.com/sql/docs/postgres/connect-kubernetes-engine
+        # TODO: use proper environment specific instance name
+        - name: cloudsql-proxy
+          image: gcr.io/cloudsql-docker/gce-proxy:1.11
+          command: ["/cloud_sql_proxy",
```

Comment:
This contains stuff that will change, so it should probably go in a rendered `entrypoint.sh` file instead of directly in the k8s definition.